### DISCUSSION TITLE
"Show menu bar items when trigger conditions are met" Forthcoming ?

### DISCUSSION THREAD

**Discussion by macmanjimmy** [jordanbaird/Ice#390] [Q&A] [answered]
Read in the app features " Show menu bar items when trigger conditions are met" but not checked, assuming this will be a forthcoming feature? If it is, that is all this app needs to be fully completed. 
I used it, in another app, for battery level, and when the battery reached below 25% it was shown an excellent feature, one I hope the "ICE" app will have, you'll have my full support when this happens :-) great app, thanks

**Comment by jordanbaird** [comment, marked as answer]
Yes, the feature is forthcoming.
